pkg/auth/ldaputil/client.go

```diff
@@ -8,10 +8,11 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 
 	"github.com/go-ldap/ldap"
+	"github.com/openshift/origin/pkg/auth/ldaputil/ldapclient"
 )
 
-// NewLDAPClientConfig returns a new LDAPClientConfig
```

Comment:
I was hoping to avoid changes like this and keep this PR limited to isolated prune changes